### PR TITLE
test: disable known-flaky test in test/pg-cdc/pg-cdc.td

### LINKED ISSUE #24357: error: expected slot materialize_% to be inactive, is active
### What version of Materialize are you using?

5ec8eb4d8d53

### What is the issue?

Seen in Postgres CDC tests in https://buildkite.com/materialize/tests/builds/72831#018cf6cd-5a22-4c36-8192-cf50fa7eccd7
```
^^^ +++
pg-cdc.td:152:1: error: expected slot materialize_% to be inactive, is active
     |
  14 | > CREATE SECRET pgpa ... [rest of line truncated for security]
  20 |     PASSWORD SECRET  ... [rest of line truncated for security]
  29 |     PASSWORD SECRET  ... [rest of line truncated for security]
  34 | $ postgres-execute c ... [rest of line truncated for security]
 151 | #
 152 | $ postgres-verify-sl ... [rest of line truncated for security]
     | ^
```
I haven't seen this failure before.

ci-regexp: to be inactive, is active
ci-apply-to: pg-cdc

### PR DESCRIPTION
**Pull request by sploiselle**
We can't yet fix #24357, so disable the test for the time being.

### Motivation

This PR mitigates the CI impact of a recognized bug.

### Checklist

- [ ] This PR has adequate test coverage / QA involvement has been duly considered.
- [ ] This PR has an associated up-to-date [design doc](https://github.com/MaterializeInc/materialize/blob/main/doc/developer/design/README.md), is a design doc ([template](https://github.com/MaterializeInc/materialize/blob/main/doc/developer/design/00000000_template.md)), or is sufficiently small to not require a design.
  
- [ ] If this PR evolves [an existing `$T ⇔ Proto$T` mapping](https://github.com/MaterializeInc/materialize/blob/main/doc/developer/command-and-response-binary-encoding.md) (possibly in a backwards-incompatible way), then it is tagged with a `T-proto` label.
- [ ] If this PR will require changes to cloud orchestration or tests, there is a companion cloud PR to account for those changes that is tagged with the release-blocker label ([example](https://github.com/MaterializeInc/cloud/pull/5021)).
  
- [ ] This PR includes the following [user-facing behavior changes](https://github.com/MaterializeInc/materialize/blob/main/doc/developer/guide-changes.md#what-changes-require-a-release-note):
  -

### CODE REVIEW

File: test/pg-cdc/pg-cdc.td [hunk 1]
```diff
@@ -148,7 +148,8 @@ CREATE PUBLICATION another_publication FOR TABLE another_schema.another_table;
 
 #
 # Test that slots created for replication sources are deleted on DROP
-$ postgres-verify-slot connection=postgres://postgres:postgres@postgres slot=materialize_% active=false
+# TODO: enable once we land https://docs.google.com/document/d/19i1lBaTKY4-G9Vi2_2-FpRzA1RecYuOCcOk0KMnDacI/edit#heading=h.a3wek7c0fc48
```
Comment (nrainer-materialize) [inline test/pg-cdc/pg-cdc.td:151]: Can we please link an issue? It will allow us to be notified when the issue is closed and the TODO is still present.

Comment (sploiselle) [reply test/pg-cdc/pg-cdc.td]: done

**Comment by sploiselle** [comment]
I took the liberty of adding some logging that would have helped me suss out this issue more quickly, as well as removing a stray comment that shouldn't have been committed.

**Review by nrainer-materialize — APPROVED** [review]
Thank you!

**Review by def- — APPROVED** [review]

**Pull request merged**